model/src/main/java/org/projectnessie/api/TreeApi.java

```diff
@@ -233,7 +233,19 @@ LogResponse getCommitLog(
           Integer maxRecords,
       @Parameter(description = "pagination continuation token, as returned in the previous LogResponse.token")
       @QueryParam("pageToken")
-          String pageToken)
+          String pageToken,
+      @Parameter(description = "The author of a commit. This is the original committer")
```

Comment:
Should mention that `null` means no filtering on the author.
(And similar for the other new parameters)

Reply:
+1 I believe the `Parameter` openapi arg has a default value flag?

Reply:
I updated the description to mention that `null` means that we're not filtering based in this value. @rymurr what did you have in mind with a default value?

Reply:
Ahh, I meant that _I think_ you can do `@Parameter(description="blah", default="empty, not filtering")`

I think this should be switched to a `List<String>` though. Empty list means return all author/committers (ie no filtering) and each entry in the list is a author in the set of authors to filter and return.We should make the entries in the List interpretable as regexs but maybe not in this patch